test/built-ins/Object/Assign/Target-Undefined-Null.js

```diff
@@ -0,0 +1,21 @@
+/*---
+description: Test the first argument(target) of Object.Assign(target,...sources),
+             if target is Undefined or null,Should Throw a TypeError exception.
+es6id:  19.1.2.1.1
+author: Wei Guo
+---*/
+
+function test(target)
+{
+    try
+    {
+        Object.assign(target,{a:1});
+        $ERROR("Should throw an exception but no error was throw");
+    }
+    catch(e)
+    {
+        assert(e.name === "TypeError" , "Expected to throw a TypeError but got " + e.name + " instead");
+    }
+}
+test(undefined);
```

Comment:
Actually, we're trying to move away from using the `negative` frontmatter for runtime errors like this. This preference is soon-to-be-documented in the project's CONTRIBUTING.md file; see gh-289.